README.md

```diff
@@ -275,8 +275,7 @@ Each group consists of:
 - a mapping of InSpec attribute names to Terraform output variable
   names to define for the suite's InSpec profile. By default, all Terraform
```

Comment:
How about:

"By default, an attribute with an equivalent name will be defined for each output variable; this mapping allows for additional or overridden associations."